newsfragments/2699.misc.rst

```diff
@@ -0,0 +1 @@
+Remove IndisputableEvidence
```

Comment:
I think there is a removal newgragment category

Reply:
That's right, it's not mentioned in README, but it is allowed by the validator. Fixed.